public/wp-content/themes/newsup-child/single-anime.php

```diff
@@ -138,9 +138,10 @@
 <!--                    print associated anime reviews in another card-->
                   <?php
                   $query_all = array(
-                      'posts_per_page' => -1,
+                      'posts_per_page' => 15,
```

Comment:
Is it 15 or is it 10?

Reply:
when i use 10, it brings up a random range, so i decided 15 would be good. even tho its on 15, it brings up 9-11 or so